tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -294,6 +294,14 @@ spec:
                 print(json_output)
 
         def main():
+
+            with open(r"updated_files.txt'", 'r') as ufiles:
```

Comment:
Why use raw strings here? This can be a normal string. But more importantly, there seems to be a redundant `'`, so the string should probably be `"updated_files.txt"`

Reply:
fixed in https://github.com/redhat-appstudio/release-service-catalog/pull/260